example_configs/tomcat.json

```diff
@@ -0,0 +1,44 @@
+{
+  "rules": [
+    {
+      "pattern": "Catalina<type=GlobalRequestProcessor, name=\"(\w+-\w+)-(\d+)\"><>(\w+):",
+      "name": "tomcat_$3_total",
+      "labels": {
+        "protocol":"$1",
+        "port":"$2" },
+      "help": "Tomcat global $3",
+      "type": "COUNTER",
+      "attrNameSnakeCase": false
```

Comment:
You'll want to use one of the options to lower case everything, as snake case is the standard Prometheus way to do things